pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java

```diff
@@ -137,11 +137,10 @@ public void init() throws DBException {
           }
         }
 
-        if (path == null) {
-          throw new DBException(PATH_PROPERTY + " is obligatory to run PmemKV client");
-        }
-        if (size == null) {
-          throw new DBException(SIZE_PROPERTY + " is obligatory to run PmemKV client");
+        if (path.isEmpty()) {
```

Comment:
what if it is specified in json config? maybe we should not print any warning here and just let pmemkv return proper errormsg?

Reply:
In Java you have object reference everywhere, also here.
Do you remember this? https://youtu.be/4V2C0X4qqLY?t=64 
You will get NullReferenceException when invoking isEmpty() method due to:
`The method returns null if the property is not found.`
https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#getProperty(java.lang.String)

Please change it to the following condition:
```java
if (!path || path.isEmpty()) {
```
Please apply it also for the `size`, there also we can set an empty (not null) String.

Reply:
as stated in some other comment - I made this possible. Errors from pmemkv are quite alright 😉 

@KFilipek, path was by default set to empty string (few lines above), so it was never a nullptr; anyway, changed it a little.